src/AnisoPairEvaluatorTwoPatchMorse.h

```diff
@@ -2,37 +2,59 @@
 // Copyright (c) 2021-2024, Auburn University
 // Part of azplugins, released under the BSD 3-Clause License.
 
-/*!
- * \file AnisoPairEvaluatorTwoPatchMorse.h
- * \brief Defines the aniostropic pair force evaluator class for Two-patch Morse potential
- */
-
 #ifndef AZPLUGINS_ANISO_PAIR_EVALUATOR_TWO_PATCH_MORSE_H_
 #define AZPLUGINS_ANISO_PAIR_EVALUATOR_TWO_PATCH_MORSE_H_
 
 #include "AnisoPairEvaluator.h"
```

Comment:
The `<string>` header should be included since we use it. It is working now because it comes from somewhere else, but it's a good idea to "include what you use" if possible.
```suggestion
#ifndef __HIPCC__
#include <string>
#endif // __HIPCC__

#include "AnisoPairEvaluator.h"
```